lib/vmpooler/pool_manager.rb

```diff
@@ -455,6 +455,31 @@ def _check_snapshot_queue
       end
     end
 
+    def migrate_vm(vm, pool)
+      Thread.new do
+        _migrate_vm(vm, pool)
+      end
+    end
+
+    def _migrate_vm(vm, pool)
+      $redis.srem('vmpooler__migrating__' + pool, vm)
+      vm_object = $vsphere[pool].find_vm(vm) || $vsphere[pool].find_vm_heavy(vm)
```

Comment:
Again (see below), not a priority to fix here, but if it's easy, this incantation could become a method with a name (e.g., `find_vsphere_pool_vm(pool, vm)`).